WooCommerce/Classes/ViewRelated/CardPresentPayments/ReceiptViewController.swift

```diff
@@ -8,9 +10,19 @@ final class ReceiptViewController: UIViewController {
     @IBOutlet private weak var webView: WKWebView!
 
     private let viewModel: ReceiptViewModel
+    private let countryCode: String
```

Comment:
It feels more natural to me that these two properties reside in the view model (`countryCode` is already there). We could even implement the track in the view model so it can be testable :) 
On the other hand, I can understand that the code would be more straightforward and less convoluted if we just keep the data and track in the view controller, so this is not a blocker 🚀  (at least we could make view model `countryCode` public and access it from there to avoid having two sources of truth 👍

Reply:
I wasn't a fan of this either, and I refactored this part a bit in https://github.com/woocommerce/woocommerce-ios/pull/6833/commits/c69bc53706016335b2a620ac80e428626808d07e - lemme know what you think!